nuxt.config.js

```diff
@@ -85,6 +85,13 @@ export default {
 			families: ["Fira+Sans:400,400i,500,600,700&display=swap"]
 		}
 	},
+	pwa: {
+		manifest: {
+			name: config.appTitle,
+			short_name: config.appShortTitle,
+			description: config.appDescription
+		}
+	},
```

Comment:
Should there be manifest entries for [`icons`](https://developer.mozilla.org/en-US/docs/Web/Manifest/icons) and [`lang`](https://developer.mozilla.org/en-US/docs/Web/Manifest/lang). Are these populated by another means?

`icons` is a mandatory field. I see that we have static/icon.png but I don't see it used anywhere. Was it meant to be added for this?

`lang` isn't mandatory. It looks like it specifies the language of the text used for things like the name, short_name and description. I'm not sure how necessary this is for the PWA though.

Reply:
The `icons` are generated automatically if `static/icon.png` is in place (see https://pwa.nuxtjs.org/modules/icon.html).

`lang` defaults to `en` which is fine.